spec/form/work_form_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe Sufia::Forms::WorkForm do
```

Comment:
That's up to you. I don't think there's a hard rule for how to test abstract classes.